using-mobile-extensions/mobile-core/signal/signal-event-reference.md

```diff
@@ -0,0 +1,95 @@
+        "templatebody"  : "{\"jsonkey\":\"jsonvalue\",\"sdkkey\":\"{%~sdkver%}\"}",
+        "contenttype"   : "application/json",
+        "timeout"       : 5
+    }
+}
+```
+
+#### Event data example
+
+Open URL Request with a deeplink URL
+
+```
+{
+    ``"id"`        `: ``"48181acd22b3edaebc8a447868a7df7ce629920a"``,
+    ``"type"`      `: ``"url"``,
+    ``"detail"`    `: {
+        ``"url"` `: ``"myApp://HomePage"
+    ``}
+}
+```
+
+
+
+### Configuration Response Content
+
+This event is dispatched on the event hub when a configuration change is processed.  If a change occurs, the listener updates the privacy status. 
+
+#### Event details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+#### Data payload definition
+
+The Signal extension will read the following keys from the configuration event:
+
+| Key            | Friendly name | Type                                                         | Optional | Description                                                  |
+| -------------- | ------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ |
+| name           | Event Name    | string                                                       | No       | Name of the event.                                           |
+| type           | Event Type    | string                                                       | No       | Type of the event.                                           |
+| source         | Event Source  | string                                                       | No       | Source of the event.                                         |
+| id             | Event ID      | string                                                       | No       | ID used for the event.                                       |
+| eventData      | Event Data    | [Event Data](https://wiki.corp.adobe.com/display/ADMSMobile/Event+Data) | No       | A map of key-value pairs that represents the configuration for all the modules. |
+| global.privacy | String        | String                                                       | No       | Contains the default setting that the user has request with regards to data privacy. |
+| timestamp      | Time Stamp    | number                                                       | No       | Time (in seconds) since Jan 1, 1970.                         |
+
+#### Event data example
+
+For a configuration change event:
+
+```javascript
+{
+    "global.privacy": "true"
+}
+```
+
+## Events dispatched and Shared State
+
+The Signal Extension does not dispatch an event and also it doesn't share any shared state.
```

Comment:
- Please lower case "Extension" --> "extension" and reword the sentence as follows:
"The Signal extension does not dispatch an event and does not share any shared state."